src/deploy/hash_files.js

```diff
@@ -5,28 +5,25 @@ const pump = promisify(require('pump'))
 
 const { hasherCtor, manifestCollectorCtor, fileFilterCtor, fileNormalizerCtor } = require('./hasher_segments')
 
-const hashFiles = async (dir, configPath, opts) => {
-  opts = {
-    concurrentHash: 100,
-    assetType: 'file',
-    statusCb: () => {},
-    ...opts,
-  }
-
-  if (!opts.filter) throw new Error('Missing filter function option')
-  const fileStream = walker([configPath, dir], { filter: opts.filter })
-  const filter = fileFilterCtor()
-  const hasher = hasherCtor(opts)
-  const fileNormalizer = fileNormalizerCtor(opts)
+const hashFiles = async (
+  dir,
+  configPath,
+  { concurrentHash, hashAlgorithm = 'sha1', assetType = 'file', statusCb, filter },
```

Comment:
`concurrentHash` and `statusCb` default values are assigned by the calling function.